cookbook/metadata.rb

```diff
@@ -0,0 +1,11 @@
+name             'baragon'
+maintainer       'EverTrue, Inc.'
+maintainer_email 'eric.herot@evertrue.com'
```

Comment:
Wouldn't the maintainer technically be Hubspot, not us, since they “own” Baragon? Just curious.